_notices/rsn0038.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 38 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Conda packages of libcumlprims, libcugraphops, and pylibcugraphops are moving to the rapidsai channel"
```

Comment:
This title is too long, and it changes the way the table renders. I am shortening it:
```suggestion
title: "Moving libcumlprims, libcugraphops, pylibcugraphops to the rapidsai channel"
```